### PR TITLE
DOCS-16171: Adds 100.7.2 changelog

### PR DESCRIPTION
**Pull request by mdb-ashley**
## DESCRIPTION
Adds 100.7.2 changelog.

## STAGING
https://docs-mongodbcom-staging.corp.mongodb.com/database-tools/docsworker-xlarge/DOCS-16171-tools-release-100.7.2/release-notes/database-tools-changelog/

## JIRA
https://jira.mongodb.org/browse/DOCS-16171

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=647e556d67a273b816940acb

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/includes/changelogs/releases/100.7.2.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.. _100.7.2-changelog:
+
+100.7.2 Changelog
+-----------------
+
+*Released 2023-05-30*
+
+This release fixes an issue with installing Database Tools on RHEL aarch64 architecture.
```
Comment (jason-price-mongodb) [inline source/includes/changelogs/releases/100.7.2.rst:8]: I think this should be "AArch64". And, "the RHEL AArch64 ..."

File: source/includes/changelogs/releases/100.7.2.rst [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+-----------------
+
+*Released 2023-05-30*
+
+This release fixes an issue with installing Database Tools on RHEL aarch64 architecture. 
+
+Bug
+~~~
+
+- :issue:`TOOLS-3309` Fix release issue on RHEL ARM architecture
```
Comment (jason-price-mongodb) [inline source/includes/changelogs/releases/100.7.2.rst:13]: Should it be "AArch64" to match line 8?

**Review by jason-price-mongodb — APPROVED** [review]
@mdb-ashley LGTM but with questions and comments.

File: source/includes/changelogs/releases/100.7.2.rst (hunk 2 above)
Comment (mdb-ashley) [reply source/includes/changelogs/releases/100.7.2.rst:13]: I'll leave this question open for the external reviewer.

File: source/includes/changelogs/releases/100.7.2.rst (hunk 1 above)
Comment (mdb-ashley) [reply source/includes/changelogs/releases/100.7.2.rst:8]: Fixed.

File: source/includes/changelogs/releases/100.7.2.rst (hunk 2 above)
Comment (tdq45gj) [reply source/includes/changelogs/releases/100.7.2.rst:13]: I agree. I'll change TOOLS-3309 accordingly.

**Review by tdq45gj — APPROVED** [review]

File: source/includes/changelogs/releases/100.7.2.rst (hunk 2 above)
Comment (mdb-ashley) [reply source/includes/changelogs/releases/100.7.2.rst:13]: Fixed here as well. Thanks!

**Pull request merged**